### PR TITLE
2.5.0 silent swagger importer

### PR DESCRIPTION
**Pull request by olensmar**

### CODE REVIEW

File: src/main/groovy/com/smartbear/swagger/PluginConfig.java [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```
Comment (spindelmanne) [inline src/main/groovy/com/smartbear/swagger/PluginConfig.java]: Do we want the Apache license for this?

File: src/main/groovy/com/smartbear/swagger/SilentImportMethodRetriever.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.smartbear.swagger;
+
+/**
+ * The getImportMethod() must be defined in an interface, or it will not be accessible to the Ready! API plugin framework.
+ */
+public interface SilentImportMethodRetriever {
```
Comment (spindelmanne) [inline src/main/groovy/com/smartbear/swagger/SilentImportMethodRetriever.java]: Isn't this unused now?

Comment (olensmar) [inline src/main/groovy/com/smartbear/swagger/SilentImportMethodRetriever.java]: you tell me :-) Should I remove it?

Comment (spindelmanne) [inline src/main/groovy/com/smartbear/swagger/SilentImportMethodRetriever.java]: I removed it myself.

**Comment by spindelmanne** [comment]
@olensmar, could you just confirm that we're using the Apache license for this code - then I'll merge.

**Comment by olensmar** [comment]
Yes - Apache license!

**Pull request merged**